src/fuse16.rs

```diff
@@ -72,6 +72,14 @@ impl Filter<u64> for Fuse16 {
     fn len(&self) -> usize {
         self.fingerprints.len()
     }
+
+    #[cfg(feature = "analysis")]
+    type N = u16;
+
+    #[cfg(feature = "analysis")]
+    fn fingerprints(&self) -> &[u16] {
+        &self.fingerprints
+    }
```

Comment:
shouldn't it be [N] instead of u 16 since you declared it above

Reply:
they're the same so it doesn't matter semantically, but I'll change it. Thanks.